### ISSUE TITLE
list clusters name parameter acts like a wildcard

### ISSUE THREAD

**Issue by PaulAngus** [apache/cloudstack-cloudmonkey#109]
When passing a name to `list clusters` , the name is treated as if there is a wildcard on the end.
Attempting to determine the ID of a cluster as an alternative isn't possible as one cannot use `list clusters` to find the relevant `clusterid`

```
cmk list clusters filter=name name=Pool1
{
  "cluster": [
    {
      "name": "Pool1"
    },
    {
      "name": "pool10"
    }
  ],
  "count": 2
}
```

**Comment by DaanHoogland** [comment]
@PaulAngus If you test 4.16.1 can you check this? it is fixed in the backend by apache/cloudstack#5782 (and close if appropriate)

**Comment by rohityadavcloud** [comment]
This can't be fixed in the client, since 4.16.1 PR has fixed this in the API this should be fixed with 4.16.1.0 onwards. I think the issue can be closed.

**Comment by rohityadavcloud** [comment]
Fixed by https://github.com/apache/cloudstack/pull/5782 can't fix this for older ACS version as the API param is handled by mgmt server (not by cmk/cli).
